docs/components/modeler/forms/configuration/forms-config-options.md

```diff
@@ -0,0 +1,44 @@
+---
+id: forms-config-options
+title: Options Source
+description: How to configure an options source on a form field
+---
+
+## Configuring form field options sources
+
+Certain form fields work from a set of pre-populated options that your user will want to select from, such as the [Radio](../form-element-library/forms-element-library-radio.md) or [Select](../form-element-library/forms-element-library-select.md) fields. This source can be configured several ways, as described below.
+
+### Static :: Configured on the form schema
+
+The options will be defined directly on your form schema, the only way to modify them in the future will be to change the form definition itself.
+
+The static options group will appear to allow the configuration of the individual options.
+
+![Static Options Group Image](../assets/static-options-example.png)
+
+For each of these options, a unique `value` corresponding to the form submitted data must be provided, as well as a `label` which will be displayed to the user filling in the form. You may configure as many options as you want by using the add (+) button in the group header.
+
+### Input data :: Driven by process data
+
+The options will be mapped from form variables, similarly to how [Form field data binding](./forms-config-data-binding.md) works. Here, the `Input values key` property within the `Dynamic Options` configuration group is used to set which form variable to use as a source.
```

Comment:
Rather than say "will be" I would say "The options **are** mapped from form variables..." or "The options **are then** mapped from form variables..." -- all about active voice.